app/models/media_object.rb

```diff
@@ -35,6 +35,7 @@ class MediaObject < ActiveFedora::Base
 
   after_create :after_create
   before_save :normalize_desc_metadata!
+  before_save :update_dependent_properties!
```

Comment:
It was happening after the yield, but the yield was on the child object in `MasterFile.around_save`. I moved the callback to `MediaObject.before_save`. I'm still trying to reproduce this on my laptop.